crates/swc_plugin_runner/src/lib.rs

```diff
@@ -96,14 +97,42 @@ fn load_plugin(plugin_path: &Path, cache: &mut Option<PluginCache>) -> Result<In
 
     return match module {
         Ok(module) => {
-            let import_object = imports! {};
+            let memory = Memory::new(&wasmer_store, MemoryType::new(1, None, false))?;
+            let import_object = imports! {
+                "env" => {
+                    "memory" => memory
+                }
+            };
 
             Instance::new(&module, &import_object).context("Failed to create plugin instance")
         }
         Err(err) => Err(err.into()),
     };
 }
 
+fn copy_memory_to_instance(instance: &Instance, program: &Program) -> Result<(i32, u32), Error> {
+    let alloc = instance.exports.get_native_function::<u32, i32>("alloc")?;
```

Comment:
this may not be required, instead may need to use https://docs.rs/wasmer/2.0.0/wasmer/struct.Memory.html#method.grow instead. I'll double check later.